products/appengine/api.yaml

```diff
@@ -440,3 +440,36 @@ objects:
             name: 'shell'
             description: |
               The format should be a shell command that can be fed to bash -c.
+  - !ruby/object:Api::Resource
+    name: 'UrlDispatchRule'
+    description: |
+      Rules to match an HTTP request and dispatch that request to a service.
+    base_url: ''
+    create_url: 'apps/{{project}}?updateMask=dispatch_rules'
+    create_verb: :PATCH
+    update_url: 'apps/{{project}}'
```

Comment:
you'll need updateMask=dispatch_rules on update_url and to set delete_url as well

Reply:
added those..